modules/elements-color-picker/assets/js/editor/commands/exit-preview.js

```diff
@@ -0,0 +1,12 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class ExitPreview extends CommandBase {
+	apply( args ) {
+		if ( ! this.component.currentPicker.initialColor ) {
```

Comment:
What if it was a global color? so the `initialColor` suppose to be empty.

Reply:
Explanation?
I mean, it works with global color as intended. `currentPicker.initialColor` is not empty in this case.

Reply:
But now it set it locally. it's not the expected behavior. it should return to the prev state that the local color is empty.